app/mailers/can_order_devices_mailer.rb

```diff
@@ -100,6 +100,7 @@ def notify_support_school_can_order_but_no_one_contacted
 private
 
   def tracked_template_mail(message_type, template_id, mail_params = {})
+    logger.info "--- LOCAL DEBUG: #{self.class.name}.#{message_type}.with(#{params})" if ENV['LOCAL_DEBUG']
```

Comment:
Do we want to keep this?

Reply:
It's not ideal, I know, but it is so useful when analyzing the behavior of each test case...! 
It prints exactly the data and/or emails sent to CC